persim/__init__.py

```diff
@@ -5,5 +5,10 @@
 from .heat import *
 from .gromov_hausdorff import *
 from .visuals import *
+from .pers_landscape_exact import *
+from .pers_landscape_approx import *
+from .pers_landscape_transf import *
```

Comment:
exact and approx make sense, but id k what `transf` and `aux` are for. Could you spell the whole words out here and in the filenames?

Reply:
`transf` for transformer, where we implement scikit-learn style transformers, and `aux` for auxiliary. Should the transformers stay in their own file or should they move into their class files?